sys/include/net/loramac.h

```diff
@@ -460,26 +460,30 @@ extern "C" {
 #ifndef CONFIG_LORAMAC_DEFAULT_MIN_RX_SYMBOLS
 #define CONFIG_LORAMAC_DEFAULT_MIN_RX_SYMBOLS   (12)
 #endif
-/** @} */
 
 /**
- * @brief   Default adaptive datarate ACK limit (in s)
+ * @brief   Default adaptive datarate ACK limit
  *
- * @note This feature is not yet supported.
+ * `ADRACKReq` bit (FOpts) is set to 1 once `ADR_ACK_CNT` reach
```

Comment:
This description is too low level. This should indicate what's the ACK limit is IMO. The user has no way to know what ADR_ACK_CNT is

Reply:
ping